docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_6.md

```diff
@@ -0,0 +1,31 @@
+# Audit of partially sharing storage Data access 
+
+Test verifies that any access to the partially sharing in the object storages data is logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites 
+2. Create storage `storage1` and give only ***READ*** permission for non-admin user from the prerequisites
+3. In the `storage1`
+    - Add `file1` and `file2` to the storage root
+    - Create `folder1` with 2 files (`inner_file1` and `inner_file2`)
+    - Create `folder2` with 2 files (`inner_file3` and `inner_file4`)
+
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the **Library** page | |
+| 2 | Open the object storage `storage3` created at step 2 of the Preparations | |
```

Comment:
Fox the name of the object storage to `storage1`

Reply:
Done